web/README.md

```diff
@@ -29,7 +28,13 @@ However if you are using deployed instances of the Platform APIs then having ini
 #### DfE Sign-in (DSI) authentication
 Having initialised secret storage, in a console window:
 1. Navigate to `EducationBenchmarking.Web` project root
-2. Run the follow to create XXXX secret: `dotnet user-secrets set "XXXX" "xxxxx"`
+2. Set XXXX user secret: `dotnet user-secrets set "XXXX" "xxxxx"`
+
+#### Session cache
```

Comment:
If this is a breaking addition, please can these instructions be highlighted in the PR description?

Reply:
Excellent suggestion - is it worth tagging the commit when a local config change is needed?

Reply:
Or just pinging the `#dfe-tech-team` channel?